RELEASE-NOTES.txt

```diff
@@ -1,8 +1,13 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
-23.4
+23.5
```

Comment:
I think we don't need to define the next version, AFAIK this change is made as part of the release process of the app.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/19276/commits/4c5ff02ff68911bda2c45ebaa32c2ca1b5dbeb92.